.vscode/settings.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Can you remove this file? We do not want to include local vscode settings in the repo. They might clash with other developer's environments.